adapters/roulax/roulax.go

```diff
@@ -0,0 +1,138 @@
+package roulax
+
+import (
+	"encoding/json"
+	"net/http"
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"strings"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+
+
+
+
+// Builder builds a new instance of the Roulax adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+
```

Comment:
Please remove unnecessary line space. Just one liner should be fine to distinguish between two code blocks.